sdk/src/main/java/io/opentelemetry/sdk/metrics/view/Aggregations.java

```diff
@@ -0,0 +1,185 @@
+   * sum, count, histogram, and sum of squared deviations.
+   *
+   * <p>The boundaries for the buckets in the underlying histogram needs to be sorted.
+   *
+   * @param bucketBoundaries bucket boundaries to use for distribution.
+   * @return a new {@code Distribution}.
+   * @since 0.1.0
+   */
+  public static Aggregation distributionWithExplicitBounds(Double... bucketBoundaries) {
+    return new Distribution(bucketBoundaries);
+  }
+
+  /**
+   * Aggregation that calculates the last value of all recorded measurements.
+   *
+   * @since 0.1.0
+   */
+  public static Aggregation lastValue() {
+    return LastValue.INSTANCE;
+  }
+
+  @Immutable
+  private enum Sum implements Aggregation {
+    INSTANCE;
+
+    @Override
+    public AggregatorFactory getAggregatorFactory(InstrumentValueType instrumentValueType) {
+      return instrumentValueType == InstrumentValueType.LONG
+          ? LongSumAggregator.getFactory()
+          : DoubleSumAggregator.getFactory();
+    }
+
+    @Override
+    public Type getDescriptorType(
+        InstrumentType instrumentType, InstrumentValueType instrumentValueType) {
+      switch (instrumentType) {
+        case COUNTER_MONOTONIC:
+        case MEASURE_ABSOLUTE:
+        case OBSERVER_MONOTONIC:
+          return instrumentValueType == InstrumentValueType.LONG
+              ? Type.MONOTONIC_LONG
+              : Type.MONOTONIC_DOUBLE;
+        case COUNTER_NON_MONOTONIC:
+        case MEASURE_NON_ABSOLUTE:
+        case OBSERVER_NON_MONOTONIC:
+          return instrumentValueType == InstrumentValueType.LONG
+              ? Type.NON_MONOTONIC_LONG
+              : Type.NON_MONOTONIC_DOUBLE;
+      }
+      throw new IllegalArgumentException("This should not happen");
+    }
+
+    @Override
+    public boolean availableForInstrument(InstrumentType instrumentType) {
+      // Available for all instruments.
+      return true;
+    }
+  }
+
+  @Immutable
+  private enum Count implements Aggregation {
+    INSTANCE;
+
+    @Override
+    public AggregatorFactory getAggregatorFactory(InstrumentValueType instrumentValueType) {
+      // TODO: Implement count aggregator and use it here.
+      return NoopAggregator.getFactory();
+    }
+
+    @Override
+    public Type getDescriptorType(
+        InstrumentType instrumentType, InstrumentValueType instrumentValueType) {
+      return Type.MONOTONIC_LONG;
+    }
+
+    @Override
+    public boolean availableForInstrument(InstrumentType instrumentType) {
+      // It is not available for Observer instruments.
+      return instrumentType != InstrumentType.OBSERVER_MONOTONIC
+          && instrumentType != InstrumentType.OBSERVER_NON_MONOTONIC;
+    }
+  }
+
+  @Immutable
+  private static final class Distribution implements Aggregation {
+    private final AggregatorFactory factory;
+
+    Distribution(Double... bucketBoundaries) {
+      Utils.checkNotNull(bucketBoundaries, "bucketBoundaries");
```

Comment:
I think this is a no-op check, since the array that is created will always be non-null.

Reply:
done.